modules/va_forms/app/swagger/va_forms/v0/controller_swagger.rb

```diff
@@ -10,9 +10,8 @@ class ControllerSwagger
           security do
             key :apikey, []
           end
-
-          key :summary, 'All VA Forms'
-          key :description, 'Returns all VA Forms and their last revision date'
+          key :summary, 'Returns all VA Forms and their last revision date'
+          key :description, 'Returns an index of all available VA forms. Optionally, pass a ?query parameter to filter forms by form number or title.'
```

Comment:
"pass a ?query parameter" seems a bit odd to me. Why the `?` in the middle of the sentence?

Reply:
Yeah, probably don't need it. The question mark should be implied for a query param. I'll remove it